openmdao/recorders/case.py

```diff
@@ -591,6 +591,12 @@ def list_outputs(self,
         else:
             values = val
 
+        if isinstance(includes, str):
+            includes = [includes, ]
+
+        if isinstance(excludes, str):
+            excludes = [excludes, ]
+
```

Comment:
Does `Case.list_inputs()`  have the same problem then?

(Note that these methods are meant to mirror `list_inputs`/`list_outputs` on `System`, so the docstrings should probably match)

Reply:
I'll make the change to `Case.list_inputs()` as well. I didn't have to change anything with `System`, but it handles a `str` just fine, so I'll update its docstrings